data/json/items/tools.json

```diff
@@ -6193,6 +6193,22 @@
     }
   },
   {
+    "id": "hand_drill",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_gray",
+    "name": "hand drill",
+    "description": "A primitive manual drill with a single drill bit.  It is slow and it will exhaust you quickly.",
```

Comment:
A field with a value would make it easier to have tools that exhaust you more than other. However, I think such a change would be out of scope for this PR.